Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19-bullseye as builder
+FROM golang:1.20.1-buster as builder
```

Comment:
Thanks for catching this @mdryankov, I will remove the minor version and make the change to bullseye. If I understand correctly, the reason to use bullseye over buster is because bullseye is the codename for the latest stable Debian release, so the buster release is now old.

Please let me know if I am missing anything with this explanation. Thanks!